lua/image/renderer.lua

```diff
@@ -5,8 +5,19 @@ local magick = require("image/magick")
 local get_global_offsets = function()
   local x = 0
   local y = 0
-  if vim.opt.number then x = x + vim.opt.numberwidth:get() end
-  if vim.opt.signcolumn ~= "no" then x = x + 2 end
+
+  local numberwidth = math.max(
+      vim.opt.numberwidth:get(),
+      ('' .. vim.api.nvim_buf_line_count(0)):len()
```

Comment:
I think these are buffer-relative and this version applies the offsets for the active buffers to all the visible windows, looking into making this generic enough.